src/main/proto/ga4gh/genotype_phenotype_service.proto

```diff
@@ -20,19 +20,28 @@ import "ga4gh/genotype_phenotype.proto";
    Evidence Code Ontology (ECO): http://www.ontobee.org/browser/index.php?o=ECO
    Ontology for Biomedical Investigations (OBI): http://purl.obofoundry.org/obo/obi/browse
 */
-
 service GenotypePhenotypeService {
 
-    rpc SearchPhenotypeAssociationSets(SearchPhenotypeAssociationSetsRequest)
-        returns (SearchPhenotypeAssociationSetsResponse);
-
-    rpc SearchPhenotype(SearchPhenotypesRequest)
-        returns (SearchPhenotypesResponse);
-
-    // see SequenceAnnotations.SearchFeatureSets
-
-    rpc GetPhenotypeAssociations(SearchGenotypePhenotypeRequest)
-        returns (SearchGenotypePhenotypeResponse);
+  // Gets a list of association sets accessible through the API.
+  // `POST /phenotypeassociationsets/search` must accept a JSON version of
+  // `SearchPhenotypeAssociationSetsRequest` as the post body and will return a JSON version
+  // of `SearchPhenotypeAssociationSetsResponse`.
+  rpc SearchPhenotypeAssociationSets(SearchPhenotypeAssociationSetsRequest)
+    returns (SearchPhenotypeAssociationSetsResponse);
+
+  // Gets a list of phenotypes accessible through the API.
+  // `POST /phenotypes/search` must accept a JSON version of
+  // `SearchPhenotypesRequest` as the post body and will return a JSON version
+  // of `SearchPhenotypeAssociationSetsResponse`.
+  rpc SearchPhenotype(SearchPhenotypesRequest)
+      returns (SearchPhenotypesResponse);
+
+  // Gets a list of genotype-phenotype assoications accessible through the API.
+  // `POST /genotypephenotype/search` must accept a JSON version of
+  // `SearchPhenotypesRequest` as the post body and will return a JSON version
+  // of `SearchPhenotypeAssociationSetsResponse`.
+  rpc GetPhenotypeAssociations(SearchGenotypePhenotypeRequest)
```

Comment:
`SearchPhenotypeAssociations` would be the RPC method that accepts a `SearchPhenotypeAssociactionRequest`. `Get` is reserved for RPC methods that perform the retrieve by ID task.

The endpoints are `/phenotypes/` `/phenotypeassociationsets/`, and `/genotypephenotype/`, when the naming convention suggests `/phenotypeassociations/`. A common convention is to name the endpoint after the message name. In that case we would name it `/featurephenotypeassociations/`, which is a bit of a mouthful. I'm neutral towards which is more correct, but it is important to choose one and not mix naming schemes. If the endpoint will be named differently from the message it returns, we can at least have internal consistency in the IDL.